config/core/webhooks/domainmapping-validation.yaml

```diff
@@ -24,7 +24,7 @@ webhooks:
 - admissionReviewVersions: ["v1", "v1beta1"]
```

Comment:
We can delete these webhooks since we can merge them into the default validation and defaulting ones

Reply:
Yes if we move everything to the default ones then we can delete it. Should we? Check my comment above.